### PR TITLE
Design/updates

### PR DESCRIPTION
**Pull request by ellenvanwyk**
Updates to terminology and visual design

### CODE REVIEW

File: src/directives/templates/mmsRefList.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
                     <ul class="dropdown-menu pull-right" uib-dropdown-menu role="menu" aria-labelledby="single-button">
                         <li role="menuitem" style="padding:10px;">
                             <button class="btn btn-default btn-sm" ng-class="{'disabled': !docEditable || !runCleared}" ng-click="docMergeAction(branch)">
-                                Apply<i ng-show="!runCleared" class="fa fa-spin fa-spinner"></i>
+                                Pull In<i ng-show="!runCleared" class="fa fa-spin fa-spinner"></i>
```
Comment (shakeh) [inline src/directives/templates/mmsRefList.html]: @ellenvanwyk you decided to use "pull in" for doc merge as well?

File: src/assets/styles/components/_mms-search.scss [hunk 2]
```diff
@@ -206,6 +206,37 @@
     }
   }
 
+  /*Add light colored presentation element icons to the left of the title*/
+
```
Comment (shakeh) [inline src/assets/styles/components/_mms-search.scss]: i think we can put these in the transclude scss

File: src/directives/templates/mmsSpec.html [hunk 3]
```diff
@@ -195,8 +195,8 @@ <h2 class="prop-title">Classifiers</h2>
             <h2 class="prop-title">Location</h2>
             <span class="prop">{{element._qualifiedName}}</span>
             <h2 class="prop-title">ID&nbsp;
-                <button ng-click="copyToClipboard($event, '#spec-element-id')" class="btn btn-sm btn-default">
-                    <i class="fa fa-copy"></i>&nbsp;Copy
+                <button ng-click="copyToClipboard($event, '#spec-element-id')" class="btn btn-sm btn-default" uib-tooltip="Copy ID">
+                    <i class="fa fa-copy"></i>
```
Comment (shakeh) [inline src/directives/templates/mmsSpec.html]: looks better!

File: src/directives/templates/mmsRefList.html (hunk 1 above)
Comment (ellenvanwyk) [reply src/directives/templates/mmsRefList.html]: Yeah I iterated and it tested better

**Pull request merged**